src/app/authentication/forgot-password/forgot-password.component.ts

```diff
@@ -8,18 +12,37 @@ import { FormGroup, FormControl, Validators } from '@angular/forms';
 })
 export class ForgotPasswordComponent {
 
-  formGroup = new FormGroup({
-    password : new FormControl('', [Validators.required]),
+  fogetForm = this.fb.group({
+    userEmail: [null, [Validators.required]],
   });
 
+  constructor(
+    private fb: FormBuilder,
+    private api: ApiService,
+    private snackService: SnackService,
+    private router: Router, 
+    private spinnerService: SpinnerService,
+  ) { }
+
   hide = true;
-  getErrorMessage() {
-    const passwordControl:any = this.formGroup.get('password');
 
-    if (passwordControl.hasError('required')) {
-      return 'You must enter a value';
-    }
-    
-    return passwordControl.hasError('email') ? 'Not a valid email' : '';
+  forgetPassword() {
+    this.api.forget(this.fogetForm.value).subscribe({
+      next: (resp) => {
+        this.spinnerService.loadSpinner();
+        this.snackService.openSnackBar('Success', 1500);
+        setTimeout( () => {
+          this.spinnerService.closeSpinner();
+          this.router.navigateByUrl('session');
+        },1500);
+      },
+      error: (err) => {
+        let errMsg = err.error.message;
+        if(errMsg == "userEmail must be an email") {
```

Comment:
please create a separate 'constant' file for hardcoded values like 'userEmail must be an email'